http://dev.mutt.org/hg/mutt/rev/b7d2cb7c7ce1

# HG changeset patch
# User Daniel Jacobowitz <dan@debian.org>
# Date 1260514342 28800
# Node ID b7d2cb7c7ce19e67444412d0bc840a4e7ddf303f
# Parent  1cf34ea1f12864efa1c3f4481fc409ed6c8e11eb
Do not store header color in hcache.
If the color directive is removed from muttrc, the cached value from
the last directive would otherwise still be in effect.

--- a/hcache.c	Mon Dec 07 23:08:47 2009 -0800
+++ b/hcache.c	Thu Dec 10 22:52:22 2009 -0800
@@ -624,6 +624,7 @@
   nh.limited = 0;
   nh.num_hidden = 0;
   nh.recipient = 0;
+  nh.pair = 0;
   nh.attach_valid = 0;
   nh.path = NULL;
   nh.tree = NULL;

